src/documentation/ptransform-style-guide.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: default
+title: "PTransform Style Guide"
+permalink: /documentation/ptransform-style-guide/
+---
+
+# PTransform style guide
+
+## A style guide for writers of new reusable PTransform's
```

Comment:
Something in the headers here did not work out very well. Take a look: http://apache-beam-website-pull-requests.storage.googleapis.com/134/documentation/ptransform-style-guide/index.html

I believe roughly all headings need at least one more # -- H1 is only for the document title page.